state/protocol/protocol_state/epochs/happy_path_statemachine_test.go

```diff
@@ -129,7 +128,10 @@ func (s *ProtocolStateMachineSuite) TestBuild() {
 	require.Equal(s.T(), s.parentProtocolState.ID(), s.stateMachine.ParentState().ID(), "should not modify parent protocol state")
 
 	updatedDynamicIdentity := s.parentProtocolState.CurrentEpochIdentityTable[0].NodeID
```

Comment:
would suggest to rename for clarity:
```suggestion
	nodeIDforEjection := s.parentProtocolState.CurrentEpochIdentityTable[0].NodeID
```